File: OutliersAPP.SPA/src/app/_models/Comment.ts
import { Post } from "./Post";

export interface Comment{
    id:number;
    postId:number;
    userId:number;
    userPhotoUrl:string;
    UserName:string;
    userKnownAs:string;
    description:string;
    commentTime:Date;
    // postid:Post;
}

File: OutliersAPP.API/Models/Comment.cs
using System;
namespace OutliersAPP.API.Models
{
    public class Comment
    {
        public int Id { get; set; }
        public string description { get; set; }
        public DateTime CommentTime { get; set; }
        public Post Post { get; set; }
        public int  PostId { get; set; }
        public User User { get; set; }
        public int UserId { get; set; }
    }
}